askama/src/filters/alloc.rs

```diff
@@ -845,17 +976,49 @@ mod tests {
 
     #[test]
     fn test_indent() {
-        assert_eq!(indent("hello", 2).unwrap().to_string(), "hello");
-        assert_eq!(indent("hello\n", 2).unwrap().to_string(), "hello\n");
-        assert_eq!(indent("hello\nfoo", 2).unwrap().to_string(), "hello\n  foo");
         assert_eq!(
-            indent("hello\nfoo\n bar", 4).unwrap().to_string(),
+            indent("hello", 2, false, false).unwrap().to_string(),
+            "hello"
+        );
+        assert_eq!(
+            indent("hello\n", 2, false, false).unwrap().to_string(),
+            "hello\n"
+        );
+        assert_eq!(
+            indent("hello\nfoo", 2, false, false).unwrap().to_string(),
+            "hello\n  foo"
+        );
+        assert_eq!(
+            indent("hello\nfoo\n bar", 4, false, false)
+                .unwrap()
+                .to_string(),
             "hello\n    foo\n     bar"
         );
         assert_eq!(
-            indent("hello", 267_332_238_858).unwrap().to_string(),
+            indent("hello", 267_332_238_858, false, false)
+                .unwrap()
+                .to_string(),
             "hello"
         );
+
+        assert_eq!(
+            indent("hello\n\n bar", 4, false, false)
+                .unwrap()
+                .to_string(),
+            "hello\n\n     bar"
+        );
+        assert_eq!(
+            indent("hello\n\n bar", 4, false, true).unwrap().to_string(),
+            "hello\n    \n     bar"
+        );
+        assert_eq!(
+            indent("hello\n\n bar", 4, true, false).unwrap().to_string(),
+            "    hello\n\n     bar"
+        );
+        assert_eq!(
+            indent("hello\n\n bar", 4, true, true).unwrap().to_string(),
+            "    hello\n    \n     bar"
+        );
```

Comment:
Can you add test with string prefixes and basically all supported types in a separate test please?

Reply:
Added.